djangocms_picture/models.py

```diff
@@ -23,19 +23,38 @@
 
 # add setting for picture alignment, renders a class or inline styles
 # depending on your template setup
-PICTURE_ALIGNMENT = getattr(
-    settings,
-    'DJANGOCMS_PICTURE_ALIGN',
-    (
-        ('left', _('Align left')),
-        ('right', _('Align right')),
-        ('center', _('Align center')),
+def get_alignment():
+    alignment = getattr(
+        settings,
+        'DJANGOCMS_PICTURE_ALIGN',
+        (
+            ('left', _('Align left')),
+            ('right', _('Align right')),
+            ('center', _('Align center')),
+        )
     )
-)
+    return alignment
+
+
+# Add additional choices through the ``settings.py``.
+def get_templates():
+    choices = [
+        ('default', _('Default')),
+    ]
+    choices += getattr(
+        settings,
+        'DJANGOCMS_PICTURE_TEMPLATES',
+        [],
+    )
+    return choices
+
 
 # use golden ration as default (https://en.wikipedia.org/wiki/Golden_ratio)
 PICTURE_RATIO = getattr(settings, 'DJANGOCMS_PICTURE_RATIO', 1.6180)
 
+# required for backwards compability
+PICTURE_ALIGNMENT = get_alignment()
```

Comment:
Is this going to get deprecated in favor of the function?

Reply:
Correct, the variable is currently used in djangocms-bootstrap4